### ISSUE TITLE
[BUG]: [ERROR] Code 69 (UpdateApp): Please update your Android app.

### ISSUE THREAD

**Issue by quamt** [anidl/multi-downloader-nx#657]
### Program version

4.7.1

### Operating System

Windows

### Type

GUI

### Service

Hidive

### Command used

NIL

### Show ID

NIL

### Episode

NIL

### Console Output

```Shell
The error message appears right after selecting HiDive in the Browser GUI.


GUI server started on port 3000
[WS] Connection to public ws from '::1'
[WS] Connection from '::1'
[ERROR] Code 69 (UpdateApp): Please update your Android app.
```

**Comment by Jaynator495** [comment]
This will be fixed in the next version, since the old hidive API will be removed

**Comment by quamt** [comment]
Ok. Thank you. I'll leave this open until implementation is completed so it won't reappear.

**Comment by AnimeDL** [comment]
This should be fixed in v5.0.0 which was just released earlier, feel free to re-open if this is still an issue
